test/resources/controls/aws_parallelcluster_install/install_packages_spec.rb

```diff
@@ -0,0 +1,52 @@
+# Use the name matching the resource type
+control 'install_packages' do
+  title 'Test installation of packages'
+
+  if virtualization.system == 'docker' && os.redhat? && os[:name] == 'redhat'
```

Comment:
Please use `only_if { virtualization.system != 'docker' || os.name != 'redhat' }` as described [here](https://docs.chef.io/inspec/dsl_inspec/#use-only_if-to-exclude-a-specific-control)

Reply:
Done